packages/ffe-searchable-dropdown-react/src/getListToRender.js

```diff
@@ -1,9 +1,17 @@
+const cleanString = text =>
+    text
+        .toLowerCase()
+        .replace(/\s/g, '') // Remove all whitespace
+        .replace(/\./g, ''); // Remove all periods
+
 const filterDropdownList = (dropdownList, searchAttributes, inputValue) => {
-    const inputValueLowerCase = inputValue.toLowerCase();
+    const cleanInputValue = cleanString(inputValue);
```

Comment:
Man kunde vurdert att ha denne som en input. 

```
const inputMatches =( inputValue,  item) => boolean;
```
Hvis vi rydder vekk mellomrum og punktum så ødelegger vi hvis noen i framtiden skulle trenga dette. Jag synes en default implmentasjon slik det var er bra med mulighetern før og øverstyra

Reply:
Mellomrum kan jag kanske gå med på men punktum synes jag ikke burde bort

Reply:
Godt poeng. Da har jeg lagt på en optional prop `searchMatcher`, laget en test for den og brukt den i `SearchableDropdown.md`. Har også fiksa det andre du kommenterte, har amenda det inn i commiten :)